ui-v2/app/instance-initializers/event-source.js

```diff
@@ -0,0 +1,61 @@
+import config from '../config/environment';
```

Comment:
Below here we dynamically create the new EventSource returning repositories. Right now we just need to do this with nodes and services - other models will be added in here to enable blocking queries support for those models.